helm/cas-registration/templates/NetworkPolicy/backendPostgresIngress.yaml

```diff
@@ -11,9 +11,9 @@ spec:
   - from:
     - podSelector:
         matchLabels:
-          release: {{ .Release.Name }}
+          release: {{ include "cas-registration.fullname" . }}
```

Comment:
When Ballard, Mike and I were testing out a deploy to the funny namespace, I was installing the helm chart using the instance name `cas-bciers`. Because some of the pod/service labels (and some of the match labels are marked with `.Release.Name` and others used `...fullname`, certain things weren't being found. 
The `{{ include "...fullname" . }}` uses just the release name if it matches the chart name, but if they differ, it appends it to the name. So in Reg they all just become `cas-registration`, because that's what the instance and chart name is, but in this case, it would become `cas-bciers-cas-registration`. And because some things used just `.Release.Name`, there were a lot of missed/conflicting identifiers leading to things not finding anything. 
I went through and changed all the matchesLabels and metadata to use the `include` for consistency, because it also gets used in another `_helper.yaml` function to generate other metadata. It also allows to set an override name in the values and have that fed through the other helpers. 

When it comes to cas-registration (which I'll tripple check), the templates should render the exact same as before. But it should produce more consistent results moving forward with all the labels aligned at least.

Reply:
Makes sense! I think we should flag these changes as potentially breaking when we merge things back into develop, and be careful with it. In particular, `release: {{ include "cas-registration.fullname" . }}` makes me think we set something up wrong, if the release label needs to be something different than the release name

Reply:
Just did a compare between templated output between the `develop`-branch and this branch. Using the same `values-dev.yaml` and same `cas-registration` instance name, they turn out identical with just a couple of minor changes in metadata. I can share the comparison later, if we'd like to check into it further.

Reply:
> Makes sense! I think we should flag these changes as potentially breaking when we merge things back into develop, and be careful with it. In particular, `release: {{ include "cas-registration.fullname" . }}` makes me think we set something up wrong, if the release label needs to be something different than the release name

@pbastia I wonder if it is a matter of naming conventions that have been adopted by CAS? I went to read more about it in the Kubernetes docs to make sure I'm thinking right ([release being used for alpha/beta/etc](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#motivation), and [common labels here](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)), and it _seems_ like the release label is used in a different manor. 

Ultimately, the important part is that the labels and selectors match up in a predictable way.